indexer/ethereumetl/jobs/export_traces_job.py

```diff
@@ -50,7 +50,7 @@ def __init__(
         self.web3 = web3
 
         # TODO: use batch_size when this issue is fixed https://github.com/paritytech/parity-ethereum/issues/9822
-        self.batch_work_executor = BatchWorkExecutor(starting_batch_size=batch_size, max_workers=max_workers, job_name=type(self).__name__)
+        self.batch_work_executor = BatchWorkExecutor(starting_batch_size=1, max_workers=max_workers, job_name=type(self).__name__)
```

Comment:
tại sao lại thay đổi?

Reply:
because [https://github.com/Octan-Labs/pipeline/blob/13d07fd9d572ab40ebf176f8b44a9f6387ac7de7/indexer/ethereumetl/jobs/export_traces_job.py#L72-L103](url) gets traces of first block of blocks patch